Dist folder files is not covered in package.json `exports` field

**Environment info**

- Cytoscape.js version : 3.29.0
- Node.js & version : any Node version that supports `exports` field

**Current (buggy) behaviour**

In the current behaviour, this package was added an `exports` field to its package.json which doesn't cover all the `dist` folder consumble outputs.



**Desired behaviour**

I would be able to directly consume any dist file e.g
```
require('cytoscape/dist/cytoscape.umd')
```

**Minimum steps to reproduce**

Use the latest version and have this simple test file:
```
const test = require('cytoscape'); // works

console.log(test);

const testFail = require('cytoscape/dist/cytoscape.umd'); // Fails

console.log(testFail);
```

<img width="1419" alt="image" src="https://github.com/cytoscape/cytoscape.js/assets/49120070/290aaf40-7edd-4400-8b41-6956420bf1bd">

I would expect anything I can consume would be covered over the `exports` field 👍 

For example the following condition would be sufficient -> 
```
    "./dist/*": {
      "import": "./dist/*.js",
      "require": "./dist/*.js"
    }
```

**For reviewers**

_Reviewers should ensure that the following tasks are carried out for incorporated issues:_

- [ ] Ensure that the reporter has included a reproducible demo.  They can easily fork this JSBin demo: http://jsbin.com/fiqugiq
- [ ] The issue has been associated with a corresponding [milestone](https://github.com/cytoscape/cytoscape.js/milestones).
- [ ] The commits have been incorporated into the corresponding branches.  Bug-fix patches go on
    - [ ] `master`,
    - [ ] `unstable`, and
    - [ ] the previous feature release branch (e.g. `1.1.x` if the current release is 1.2).
- [ ] The issue has been labelled as a [`bug`](https://github.com/cytoscape/cytoscape.js/labels/bug), if necessary.

can confirm this bug, got this error with 3.29.0 - `./node_modules/mermaid/dist/mindmap-definition-44684416.js:4:0-56 - Error: Module not found: Error: Package path
./dist/cytoscape.umd.js is not exported from package`

We're running into a similar issue trying to build/deploy Docusaurus on AWS Amplify:

```
Module not found: Error: Package path ./dist/cytoscape.umd.js is not exported from package /codebuild/output/.../node_modules/cytoscape (see exports field in /codebuild/output/.../node_modules/cytoscape/package.json)
```

@mikekucera - thanks for looking into this. With `3.29.1`, I'm still getting the following error:
```
Module not found: Error: Can't resolve 'cytoscape/dist/cytoscape.umd.js'  in '/path-to-project/node_modules/mermaid/dist'
```

--- a/package.json
+++ b/package.json
@@ -43,6 +43,9 @@
     "./dist/cytoscape.esm.min": {
       "import": "./dist/cytoscape.esm.min.mjs"
     },
+    "./dist/cytoscape.umd.js": {
+      "import": "./dist/cytoscape.umd.js"
+    },
     "./dist/*": {
       "import": "./dist/*.js",
       "require": "./dist/*.js"

